pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -147,7 +148,7 @@ let
 
     pname = "neovim";
     version = lib.getVersion neovim-unwrapped;
-  in {
+  in builtins.removeAttrs attrs ["extraPython3Packages" "extraLuaPackages" ] // {
```

Comment:
Is that related to enabling the check phase ?

Reply:
instead of adding `inherit doCheck` I thought I could remove all the `inherit withPython, etc` by directly reusing the removeAttrs. Except I haven't removed the other `withXX` so it's a bit redundant :/ Had to remove everything that is a function from attrs since nix cant serialize it

Reply:
Ok I see... Feels a bit hacky no ?

Reply:
that's how most builders work so I am not sure. It's a style tradeoff. Anyhow I've reverted it since it seems confusing and might have some unforeseen consequences